Desktop/components/JASP/Widgets/ErrorMessage.qml

```diff
@@ -0,0 +1,100 @@
+//
+// Copyright (C) 2013-2022 University of Amsterdam
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU Affero General Public License as
+// published by the Free Software Foundation, either version 3 of the
+// License, or (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU Affero General Public License for more details.
+//
+// You should have received a copy of the GNU Affero General Public
+// License along with this program.  If not, see
+// <http://www.gnu.org/licenses/>.
+//
+import QtQuick
+import JASP
+
+//copied from ControlErrorMessage, but without all the JS
+Rectangle
+{
+	id				: errorMsg
+	color			: warning ? jaspTheme.controlWarningBackgroundColor : jaspTheme.controlErrorBackgroundColor
+	visible			: opacity > 0 && text != ""
+	opacity			: 1
+	width			: implicitWidth
+	height			: implicitHeight
+	implicitWidth	: 200 * jaspTheme.uiScale
+	implicitHeight	: messageText.contentHeight + (2 * paddingHeight) + (!dontShowAgain ? 0 : dontShowAgainButton.height + (paddingHeight * 2))
+	z				: 10
+	radius			: jaspTheme.borderRadius
+	border.color	: foreCol
+	border.width	: 1
+
+	property color foreCol: warning ? jaspTheme.controlWarningTextColor : jaspTheme.controlErrorTextColor
+
+	property bool dontShowAgain : false
+	property bool warning		: false
+	property int paddingWidth	: 10 * jaspTheme.uiScale
+	property int paddingHeight	:  6 * jaspTheme.uiScale
+	property alias wrapMode:	messageText.wrapMode
+	property alias text:		messageText.text
+	property alias textFormat:	messageText.textFormat
+
+	signal dontShowAgainClicked()
+
+	Behavior on opacity
+	{
+		enabled: preferencesModel.animationsOn;
+
+		NumberAnimation
+		{
+			duration			: 300
+			easing.type			: Easing.InOutQuad
+		}
+	}
+
+	CrossButton { onCrossClicked: {	errorMsg.opacity = 0 } }
```

Comment:
By clicking the cross button, this will just make the message disappear, but it is not removed: the place it takes will be still used, making a big empty place.

Reply:
oops